tests/neg/typeclass-encoding3.scala

```diff
@@ -344,5 +344,6 @@ object functors {
       ??? // $this.flatMap[A](identity)   disabled since it does not typecheck
   }
 
-  MonadFlatten.flattened(List(List(1, 2, 3), List(4, 5)))
+  MonadFlatten.flattened(List(List(1, 2, 3), List(4, 5))) // ok, synthesizes (using ListMonad)
+  MonadFlatten.flattened(List(List(1, 2, 3), List(4, 5)))(using ListMonad) // error
```

Comment:
To be clear: passing ListMonad explicitly already failed on master and this PR doesn't improve the situation but just documents it?

Reply:
yes, correct.